typescript/widgets/src/layout/Card.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { PropsWithChildren } from 'react';
+
+export type CardProps = PropsWithChildren<{
+  className?: string;
+  inverseMargin?: boolean;
+}>;
+
+export function Card({ className, inverseMargin, children }: CardProps) {
+  return (
+    <div
+      className={`${
+        inverseMargin ? cardStyles.inverseMargin : cardStyles.padding
```

Comment:
When would you want the inverse margins applied here? They exist in the warp ui so the notification banner can use them and be flush with the card edges. 

More broadly, I think the exist shape, shadowing, padding, and roundedness of cards are kind of an app-specific thing. I recommend we remove this for now and let the apps define their own card components, which are quite simple anyway

Reply:
Yeah makes sense to me, I will remove this and each app can have their own Card component